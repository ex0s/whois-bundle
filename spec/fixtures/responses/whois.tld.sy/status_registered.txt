Domain Name: tld.sy
Domain ID: 7-sy
WHOIS Server: ?????? ??????? ?????? ?????? - ????? ?????? ??????
Referral URL: 
Creation Date: 2010-12-02T16:01:27.664Z
Registry Expiry Date: 2017-12-01T22:00:00.000Z
Sponsoring Registrar: nans
Sponsoring Registrar IANA ID: 
Domain Status: ok

Registrant ID: 6714-sy
Registrant Email: domain@tld.sy

Billing ID: 6714-sy
Billing Email: domain@tld.sy

Name Server: ns4.tld.sy
Name Server: ns3.tld.sy

DNSSEC: unsigned




>>> Last update of WHOIS database: 2014-02-18T09:55:11.758Z <<<
