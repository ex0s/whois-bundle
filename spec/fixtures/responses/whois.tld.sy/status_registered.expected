#domain
  %s == "tld.sy"

#domain_id
  %s == "7-sy"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2010-12-02 16:01:27 UTC}

#updated_on
  %s %ERROR{AttributeNotSupported}

#expires_on
  %s %CLASS{time}
  %s %TIME{2017-12-01 22:00:00 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == nil
  %s.name         == "nans"
  %s.organization == nil
  %s.url          == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns4.tld.sy"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns3.tld.sy"
